main.py

```diff
@@ -6,10 +6,22 @@
 import pygame as pg
 from pygame.color import THECOLORS as COLORS
 from pygame.math import Vector2 as V2
+import pygame_menu as pg_menu
 
 OVERMAP_BG = COLORS["white"]
 BOARD_COLOR = COLORS["grey"]
 BACKGROUND_COLOR = COLORS["black"]
+COLOR_BLOP = COLORS["green"]
+
+PLAYER_NAME = "Player"
+
+INDEX = [
```

Comment:
Mal nommé -> `INDEX` est un mot trop générique pour comprendre ce à quoi sert cette constante :
 ```suggestion
COLOR_MAPPING = [
```

Reply:
Par ailleurs tu peux t'en sortir sans ce mapping en utilisant `str.capitalize`.
Exemple (à tester en console python) :
```python
color = "black"
print(color.capitalize())
# => "Black"
```